### DISCUSSION TITLE
LIMIT option in alerts API

### DISCUSSION THREAD

**Discussion by nikhilKipi** [weather-gov/api#652] [General]
Is there any issue with LIMIT option in alerts API? because irrespective of I apply the limit clause or not, the response still contains row count <=500 rows. Here is an example below.

Sample request without limit - https://api.weather.gov/alerts?start=2023-08-21T00%3A00%3A01Z&end=2023-08-21T23%3A23%3A59Z
[without_limit_alert.txt](https://github.com/weather-gov/api/files/12441694/without_limit_alert.txt)

Sample request with limit - https://api.weather.gov/alerts?start=2023-08-21T00%3A00%3A01Z&end=2023-08-21T23%3A23%3A59Z&limit=500
[with_limit_alert.txt](https://github.com/weather-gov/api/files/12441702/with_limit_alert.txt)

**Comment by JJKraw** [comment]
How many active alerts were there at the time you issued this?
